src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -230,18 +223,15 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
           />
           {/* move pinchZoomContainer if search is visible on mobile */}
           <div
+            id={'pinchZoomContainer'}
```

Comment:
do you need the { braces around the string?

Reply:
no, and I don't need the id as well, was just checking something :) Will remove